scripts/chat_nonvalidated_users.py

```diff
@@ -0,0 +1,38 @@
+import argparse
+import os
+
+import rows
+import tqdm
+
+from bot.rocketchat import RocketChat
+
+
+def run(output_filename):
+    rocket_chat = RocketChat(os.environ["ROCKETCHAT_BASE_URL"])
+    rocket_chat.login(
+        os.environ["ROCKETCHAT_USERNAME"],
+        os.environ["ROCKETCHAT_PASSWORD"]
+    )
+    all_users = rocket_chat.user_list()
+    with open(output_filename, "w") as csvfile:
+        writer = rows.utils.CsvLazyDictWriter(csvfile)
+
+        for user in tqdm.tqdm(all_users, desc="Buscando usuários não verificados..."):
+            email = user.get("emails")
+            if email:
+                verified = email[0]["verified"]
+                if verified is False:
+                    writer.writerow(
+                        {
+                            "username": user.get("username"),
+                            "name": user.get("name"),
+                            "email": email[0].get("address"),
+                        }
+                    )
```

Comment:
Precisamos pegar todos os emails de cada usuário, algo como:

```suggestion
            emails = user.get("emails") or []
            for email in emails:
                verified = email["verified"]
                if verified is False:
                    writer.writerow(
                        {
                            "username": user.get("username"),
                            "name": user.get("name"),
                            "email": email.get("address"),
                        }
                    )
```

Reply:
Solved by 550e330